pixiv.js.org

dragon-fish opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

GitHub repo: [FreeNowOrg/PixivNow](https://github.com/FreeNowOrg/PixivNow)

CNAME: https://github.com/FreeNowOrg/PixivNow/tree/master/CNAME

dragon-fish committed:
Update cnames_active.js

indus commented:
I've added the requested subdomain to test. But I've seen that the requests get redirect to page with content that doesn't seem to be related to JavaScript. We wouldn't support that.

dragon-fish commented:
> I've added the requested subdomain to test. But I've seen that the requests get redirect to page with content that doesn't seem to be related to JavaScript. We wouldn't support that.

Fixed

indus commented:
@dragon-fish I still don't see how this is related to JavaScript? I really don't know why this was accepted last time?!?

indus commented:
Probably I should just put it back in as a grandfathering, but you would have to add the ``` // noCF ``` at the end of your line in the list before I can merge this.

dragon-fish commented:
> @dragon-fish I still don't see how this is related to JavaScript? I really don't know why this was accepted last time?!?

Well, if there's need a reason. it's a fullstack project in TypeScript.

dragon-fish committed:
Create cnames_active.js

indus commented:
Welcome back https://pixiv.js.org

dragon-fish commented:
I just realized JS-ORG updated the rules. Thank you very much for still choosing to accept my request! ❤️
